utils/data/target_metadata/low_mass_stars_archive_CJK_JRD_STIS.csv

```diff
@@ -502,10 +502,10 @@ O5E308050,V410-TAU,64.6299167,28.4545556,20,1/30/01 15:06,1/30/01 15:50,2650.113
 #O5E309050,V819-TAU,64.85975,28.4374167,20,8/30/00 22:34,8/30/00 23:18,2650.183,STIS,0.2X0.06,E140M,1425,8157,8/30/01 23:18,O5E309050,do
 O8NS05010,V826TAU,68.0659167,18.0275278,2,9/8/03 15:01,9/8/03 15:04,162,STIS,52X0.2,G430L,4300,9790,12/16/06 09:00,O8NS05010,do
 O8NS05020,V826TAU,68.0659167,18.0275278,2,9/8/03 15:10,9/8/03 15:41,1859.191,STIS,52X0.2,G140L,1425,9790,12/16/06 13:38,O8NS05020,
-O8NS04010,V827TAU,68.060625,18.3375,2,9/7/03 15:01,9/7/03 15:03,1.536,STIS,52X0.2,G430L,4300,9790,12/16/06 02:19,O8NS04010,
-O8NS04020,V827TAU,68.060625,18.3375,2,9/7/03 15:10,9/7/03 15:10,0,STIS,52X0.2,G140L,1425,9790,12/16/06 05:26,,
-O8NS40010,V827TAU,68.060625,18.3375,2,1/26/04 01:05,1/26/04 01:09,162,STIS,52X0.2,G430L,4300,9790,12/18/06 01:29,O8NS40010,
-O8NS40020,V827TAU,68.060625,18.3375,2,1/26/04 01:15,1/26/04 01:46,1859.18,STIS,52X0.2,G140L,1425,9790,12/18/06 04:56,O8NS40020,
+O8NS04010,V827TAU,68.060625,18.3375,2,9/7/03 15:01,9/7/03 15:03,1.536,STIS,52X0.2,G430L,4300,9790,12/16/06 2:19,O8NS04010,

```

Comment:
@jduval82 it looks like excel stripped the zeros here. I don't think this will cause any problems, but are you fine with this?

Reply:
That will not be a problem (just annoying that excel would do that!)